.chloggen/31377-windowsservicereceiver-wireframe.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'new_component'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: windowsservicereceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: receiver for scraping windows service information from a host or properly configured remote machine.
```

```suggestion
note: Introduce a new receiver to retrieve Windows services status information.
```

Initially is just the wire-frame and separating the support of remote that can be added later with the caveat that we should create a config that is compatible with later adding remote without breaking the initial config. Anyway, there is still more flexibility while the component is still in development.